appender_test.go

```diff
@@ -20,7 +20,7 @@ import (
 )
 
 type simpleStruct struct {
-	A int32
+	A int32 `db:"a"`
 	B string
 }
```

Comment:
Let's add another test for a `struct` like the one below. It should return a duplicate field error.

```Go
type myStruct struct {
	A int64 `db:"Duplicate"`
	Duplicate int64 
}
```

And for completeness, another one like this:
```Go
type myStruct struct {
	a         int64 `db:"does_not_exist"`
	B int64
}
```

Reply:
I added a handling for the duplicate keys case. However I am not sure how to handle the missing key case as it seems to be unhandled even without field renaming.

Any suggestions on how to proceed?

Thanks

Reply:
Ah, sorry, I should've phrased my message better. However, it is not strictly necessary, since we already test that path later.

> And for completeness, another **error-test** like this: